test/src/test/java/hudson/tasks/EnvVarsInConfigTasksTest.java

```diff
@@ -92,10 +93,9 @@ public void testFreeStyleShellOnSlave() throws Exception {
 	}
 
 	public void testFreeStyleAntOnSlave() throws Exception {
-        if (jenkins.getDescriptorByType(Ant.DescriptorImpl.class).getInstallations().length == 0) {
-            System.out.println("Cannot do testFreeStyleAntOnSlave without ANT_HOME");
-            return;
-        }
+	    // Cannot do testFreeStyleAntOnSlave without ANT_HOME
+	    Assume.assumeTrue((jenkins.getDescriptorByType(Ant.DescriptorImpl.class).getInstallations().length > 0);
```

Comment:
`assumeThat(jenkins.getDescriptorByType(Ant.DescriptorImpl.class).getInstallations(), not(empty()))`